www/include/monitoring/downtime/AddDowntime.php

```diff
@@ -283,8 +283,30 @@
         )
     );
 
+    $durationScale = array(
+        'd' => 86400,
+        'h' => 3600,
+        'm' => 60,
+        's' => 1,
+    );
     $defaultDuration = 7200;
+    $defaultScale = 's';
+    if (isset($centreon->optGen['monitoring_dwt_duration']) &&
+        $centreon->optGen['monitoring_dwt_duration']
+    ) {
+        $defaultDuration = $centreon->optGen['monitoring_dwt_duration'];
```

Comment:
$centreon->optGen['monitoring_dwt_duration'] is not linked to this default duration.
These two parameters aren't the same. Please check my previous comments

Reply:
Yes, related to the first conversation above (https://github.com/centreon/centreon/pull/7606#discussion_r351911009).